### PR TITLE
docs: rewrite the Composition API without the module as dependency and fix the fetch

### PR DESCRIPTION
**Pull request by Dawntraoz**
docs: Update the latest changes.

fix: Alias playground error and fetch call that didn't resolve the relations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +147,54 @@ Which is the short-hand equivalent to using `useStoryblokApi` and `useStoryblokB
 </template>
 ```
 
+#### Composition API without `@nuxtjs/composition-api`
+
+> You don't need to install [@nuxtjs/composition-api](https://composition-api.nuxtjs.org/) as it's not a dependency anymore.
```
Comment (alexjoverm) [inline README.md]:
```suggestion
> You don't need to install [@nuxtjs/composition-api](https://composition-api.nuxtjs.org/) if you're in the latest versions of Nuxt 2, as it comes with Vue 2.7 with Composition API support out of the box.
```

File: README.md [hunk 2]
```diff
@@ -143,6 +147,54 @@ Which is the short-hand equivalent to using `useStoryblokApi` and `useStoryblokB
 </template>
 ```
 
+#### Composition API without `@nuxtjs/composition-api`
```
Comment (alexjoverm) [inline README.md]: I would move this section above, as I'd consider it the primary option

**Review by alexjoverm — DISMISSED** [review]

**Comment by Dawntraoz** [comment]
@alexjoverm updated ☑️

**Comment by Dawntraoz** [comment]
Hi @manuelschroederdev, as Alex is on PTO, can you double-check the updates on the Readme? 🫂

**Comment by Dawntraoz** [comment]
@alexjoverm I added the fix and the alias update in this PR. When calling `fetch()` inside the module, then the `resolve_relations` are resolved, otherwise didn't resolve them while using `script setup`.

File: README.md [hunk 3]
```diff
@@ -97,9 +104,26 @@ To link your Vue components to their equivalent you created in Storyblok:
 
 ### 2. Getting Storyblok Stories and listen to Visual Editor events
 
-#### Composition API
+#### Composition API without `@nuxtjs/composition-api`
+
+> You don't need to install [@nuxtjs/composition-api](https://composition-api.nuxtjs.org/) if you're in the latest versions of Nuxt 2, as it comes with Vue 2.7 with Composition API support out of the box.
+
+The simplest way is by using the `useStoryblok` one-liner composable, which uses the [useFetch from @nuxtjs/composition-api](https://composition-api.nuxtjs.org/lifecycle/useFetch) under the hood:
```
Comment (alexjoverm) [inline README.md]: I realize I would skip the useFetch explanation, as it might be confusing for the user

**Review by alexjoverm — CHANGES_REQUESTED** [review]

**Review by alexjoverm — APPROVED** [review]

**Pull request merged**